airflow/jobs/scheduler_job.py

```diff
@@ -28,7 +28,8 @@
 from contextlib import redirect_stderr, redirect_stdout
 from datetime import timedelta
 from itertools import groupby
-from typing import List, Optional, Set, Tuple
+from time import sleep
```

Comment:
`time` is still imported. I'd probably say don't add this nor change the `sleep` from `time.sleep`.